python-pytest.el

```diff
@@ -132,7 +132,8 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
     "                                          "] ;; visual alignment
    [("--dm" "run doctests" "--doctest-modules")
     ("--nf" "new first" "--new-first")
-    ("--sw" "stepwise" "--stepwise")]]
+    ("--sw" "stepwise" "--stepwise")]
+   [("--co" "collect only" "--collect-only")]]
```

Comment:
seems you added a new group (using `[…]`) instead of adding it below `--sw`.

not sure how that looks like? why not this?
```suggestion
    ("--sw" "stepwise" "--stepwise")
    ("--co" "collect only" "--collect-only")]]
```

Reply:
I saw that the option was in the group "collection:" (when running `pytest -h`), but then it felt overkill to have a group for only one option, and I ended up with something half-way. Sorry :)